### PR TITLE
Feature/how it's made

### PR DESCRIPTION
**Pull request by KamilaDudzik**
Feedback appreciated as we're still figuring with Gosia how to merge our two separate branches

### CODE REVIEW

File: src/sass/partials/_how-its-made.scss [hunk 1]
```diff
@@ -0,0 +1,127 @@
+
+.section {
+    padding-top: 125px;
+    padding-bottom: 145px;
+    background-color: $VBgColor;
+    font-weight: 400;
```
Comment (matilis) [inline src/sass/partials/_how-its-made.scss]: It is not necessary. The browser automatically uploads font-weight: 400;

**Review by matilis — COMMENTED** [review]
Good job, delete only font-weight: 400; in .section

**Review by matilis — APPROVED** [review]

**Review by matilis — APPROVED** [review]

File: src/sass/partials/_how-its-made.scss [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+        &__icon-list {
+            flex-direction: row;
+            gap: 55px;
+            padding: 0 16px;
+        }
+
+        &__small-list {
+            display: flex;
+            flex-direction: column;
+            gap: 16px;
```
Comment (KJachimstal) [inline src/sass/partials/_how-its-made.scss]: Wrong spaces between elements in desktop and tablet size

**Review by KJachimstal — CHANGES_REQUESTED** [review]
Missing styles for tablet and desktop screen sizes.

File: src/sass/partials/_how-its-made.scss [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+.how-its-made {
+
+    &__container {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+    }
+
+    &__title {
```
Comment (romizka) [inline src/sass/partials/_how-its-made.scss:19]: It seems to me that the title should be written in capital letters (uppercase) :)

File: src/sass/partials/_how-its-made.scss [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+        }
+    }
+
+    &__text-item {
+        display: flex;
+        flex-direction: column;
+        gap: 20px;
+    }
+
+    &__button {
```
Comment (romizka) [inline src/sass/partials/_how-its-made.scss:75]: On the mobile version the "read me" button is 14px, if I check correctly.

**Review by romizka — APPROVED** [review]

File: src/sass/partials/_how-its-made.scss (hunk 3 above)
Comment (KamilaDudzik) [reply src/sass/partials/_how-its-made.scss:19]: Oh, true! To be honest I didn't even notice it due to font's aesthetic side

File: src/sass/partials/_how-its-made.scss (hunk 4 above)
Comment (KamilaDudzik) [reply src/sass/partials/_how-its-made.scss:75]: Heh, good eye :>

**Review by KJachimstal — APPROVED** [review]

**Pull request merged**